### ISSUE TITLE
Localtunnel is down again (affecting Cogent customers & ISPs) 

### ISSUE THREAD

**Issue by Parking-Master** [localtunnel/localtunnel#576]
Localtunnel is down again

**Comment by TheBoroer** [comment]
hmm, it shouldn't be. There's currently over 1500 tunnels up. Could be an ISP issue on your end or somewhere along the line. If you dont mind me asking, where are you connecting from?

**Comment by Parking-Master** [comment]
What do you mean?

**Comment by Parking-Master** [comment]
Here’s the url: https://simplemailer.loca.lt

**Comment by TheBoroer** [comment]
It's up and working as far as i can tell. It could be your internet provider or their internet provider that's having somet temporary issues.

**Comment by Parking-Master** [comment]
Every loca.lt domain is not responding:

https://loca.lt

https://simplemailer.loca.lt

https://e.loca.lt

**Comment by TheBoroer** [comment]
those are all responding for me. Although the last two just show a "404" error...which means there's no tunnel connected with that subdomain atm.

Looks like it's a problem more on your ISP's end. just try again in a couple hours and it probably will work again :)

**Comment by Parking-Master** [comment]
Ok thanks, they’re still not responded so hopefully it’ll work later.

I figured out the second URL is showing the icon on the website tab, but it’s not loading. Very strange

**Comment by Parking-Master** [comment]
It’s not even returning an URL when I start the tunnel, are you sure it’s just me?

**Comment by CSharped** [comment]
Doesn't work for me too, running lt --port 8000 doesn't return anything

**Comment by Parking-Master** [comment]
Yeah I thought so, nothing is working for some reason. One of the strangest localtunnel issues I’ve ever seen

**Comment by TheBoroer** [comment]
Hmm this is really weird. Are you using a script to open a bunch of tunnels at one time or something that keeps retrying to open a tunnel? Might be getting rate limited. If you aren't using a script or anything then idk what it could be...

Can you run this command in your terminal and paste the output? `traceroute localtunnel.me`

**Comment by Parking-Master** [comment]
> Hmm this is really weird. Are you using a script to open a bunch of tunnels at one time or something that keeps retrying to open a tunnel? Might be getting rate limited.

No, I just use the same script I always use. It just usual starts/prints simplemailer.loca.lt but it’s not doing anything. None of the websites are loading either. It just loads forever

I am starting it from a node.js script

**Comment by TheBoroer** [comment]
Hmm yeah that's a weird issue. Can you paste the output or a screenshot of the command i sent?

**Comment by Parking-Master** [comment]
```
traceroute to localtunnel.me (193.34.76.44), 30 hops max, 60 byte packets
 1  192.168.7.1 (192.168.7.1)  2.120 ms  2.128 ms  1.947 ms
 2  134.56.126.1.hwccustomers.com (134.56.126.1)  2.939 ms  2.853 ms  4.107 ms
 3  10.96.0.53 (10.96.0.53)  4.501 ms  4.546 ms  4.365 ms
 4  10.82.0.101 (10.82.0.101)  5.553 ms  5.580 ms  5.401 ms
 5  10.96.0.22 (10.96.0.22)  12.189 ms  11.744 ms  11.558 ms
 6  170.250.254.116.hwccustomers.com (170.250.254.116)  11.940 ms  11.048 ms  11.066 ms
 7  170.250.254.5.hwccustomers.com (170.250.254.5)  12.031 ms  12.103 ms  12.651 ms
 8  * * *
 9  * * *
10  * * *
11  100ge5-1.core1.chi1.he.net (184.104.196.49)  41.667 ms 100ge16-1.core1.chi1.he.net (72.52.92.45)  41.763 ms *
12  * * *
13  * * *
14  * port-channel1.core1.tor2.he.net (184.104.197.134)  48.626 ms  48.663 ms
15  38.64.128.206 (38.64.128.206)  57.062 ms  56.715 ms port-channel1.core1.tor2.he.net (184.104.197.134)  50.504 ms
16  45.62.209.170 (45.62.209.170)  56.844 ms  57.880 ms  59.541 ms
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * * *
30  * * *
```

**Comment by Parking-Master** [comment]
@TheBoroer  Ok, here’s the output of the command:
```
traceroute to localtunnel.me (193.34.76.44), 30 hops max, 60 byte packets
 1  192.168.7.1 (192.168.7.1)  2.120 ms  2.128 ms  1.947 ms
 2  ——————.hwccustomers.com (——————)  2.939 ms  2.853 ms  4.107 ms
 3  10.96.0.53 (10.96.0.53)  4.501 ms  4.546 ms  4.365 ms
 4  10.82.0.101 (10.82.0.101)  5.553 ms  5.580 ms  5.401 ms
 5  10.96.0.22 (10.96.0.22)  12.189 ms  11.744 ms  11.558 ms
 6  170.250.254.116.hwccustomers.com (170.250.254.116)  11.940 ms  11.048 ms  11.066 ms
 7  170.250.254.5.hwccustomers.com (170.250.254.5)  12.031 ms  12.103 ms  12.651 ms
 8  * * *
 9  * * *
10  * * *
11  100ge5-1.core1.chi1.he.net (184.104.196.49)  41.667 ms 100ge16-1.core1.chi1.he.net (72.52.92.45)  41.763 ms *
12  * * *
13  * * *
14  * port-channel1.core1.tor2.he.net (184.104.197.134)  48.626 ms  48.663 ms
15  38.64.128.206 (38.64.128.206)  57.062 ms  56.715 ms port-channel1.core1.tor2.he.net (184.104.197.134)  50.504 ms
16  45.62.209.170 (45.62.209.170)  56.844 ms  57.880 ms  59.541 ms
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * * *
30  * * *
```

**Comment by TheBoroer** [comment]
Thanks @Parking-Master that was super helpful!
I talked to my provider and they confirmed there was something off with traffic routing and I think we fixed it.

Long story short, if your internet provider "peers" with Cogent (a internet backbone provider) there was a small bug in how traffic got routed to my server (and so connections timed out). Less than a handful of spots in the US had this issue (e.g. Denver, Chicago, and some others).... so you just got unlucky lol BUT thank you for letting me know! LT should be up for everyone now! 🤗 

Closing this since it should all be working. Feel free to reply if it's not and i'll re-open :)
